CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ project(rsgain
   HOMEPAGE_URL "https://github.com/complexlogic/rsgain"
   LANGUAGES CXX
 )
+
+set(WITH_UCHECKMARKS	YES	CACHE BOOL "Enable use of Unicode for checkmarks")
```

Comment:
I prefer to use an ON/OFF option for this. So, it could be titled `UCHECKMARKS`, the code would look like:
```CMake
option(UCHECKMARKS "Enable use of Unicode checkmarks" ON)
```

Reply:
Fixed